api/v1/controllers/authenticate.js

```diff
@@ -48,15 +49,26 @@ module.exports = {
           return u.handleError(next, _err, resourceName);
         }
 
-        // create token
-        const token = jwtUtil.createToken(req.user.name, req.user.name);
-        req.session.token = token;
-        const retObj = {
-          success: true,
-          message: 'authentication succeeded',
-        };
-        u.logAPI(req, resultObj, retObj);
-        return res.status(httpStatus.OK).json(retObj);
+        return Profile.isAdmin(user.profileId)
+        .then((isAdmin) => { // update in token payload if admin
+          const payloadObj = {
+            ProfileName: user.profile.name,
+            IsAdmin: isAdmin,
```

Comment:
I think it should all the other details too, (IsCollector, IsBot, etc.)

Reply:
Do we have to add it here too? I think when the token is decoded and the headers are set we will have all these information.

Reply:
Yeah, we don't need to add the details when creating token. Discussed with Ian.